website/blog/2020-07-17-react-native-principles.md

```diff
@@ -0,0 +1,53 @@
+---
+title: React Native Team Principles
+author: Eli White
+authorTitle: Engineering Manager on React Native
+authorURL: https://twitter.com/Eli_White
+authorImageURL: https://avatars2.githubusercontent.com/u/249164?s=460&v=4
+authorTwitter: Eli_White
+category: announcements
+---
+
+Our team at Facebook has principles that guide how we prioritize our work on React Native. These principles specifically represent our team and not necessarily every stakeholder in the React Native community. We are sharing these principles to be more transparent about what drives us, how we make decisions, and how we focus our efforts.
+
+## **Native Experience**
+
+Our top priority for React Native is to **match people’s expectations on each platform**. This is why React Native renders to platform primitives. We value native look-and-feel over cross-platform consistency.
+
+For example, the TextInput in React Native renders to a UITextField on iOS. This ensures that integration with password managers and keyboard controls work out of the box. By using platform primitives, React Native apps are also able to stay up-to-date with design and behavior changes from new releases of Android and iOS.
+
+In order to match the look-and-feel of native apps, we must also match their performance. This is where we focus our most ambitious efforts. For example, Facebook created Hermes, [a new JavaScript Engine built from scratch for React Native on Android](https://facebook.github.io/react-native/blog/2019/07/17/hermes). Hermes significantly improves the start time of React Native apps. We are also working on major architectural changes that optimize the threading model and make React Native easier to interoperate with native code.
+
+## Massive Scale
+
+Hundreds of screens in the Facebook app are implemented with React Native. The Facebook app is used by billions of people on a huge range of devices. **This is why** **we invest our efforts into the most challenging problems at scale.**
```

Comment:
```suggestion
Hundreds of screens in the Facebook app are implemented with React Native. The Facebook app is used by billions of people on a huge range of devices. **This is why** **we invest in the most challenging problems at scale.**
```

We've already said that these principles guide our efforts, so no need to restate